src/main/java/at/hannibal2/skyhanni/features/chat/playerchat/PlayerChatModifier.kt

```diff
@@ -21,7 +21,8 @@ class PlayerChatModifier {
     fun onChat(event: SystemMessageEvent) {
         val newMessage = cutMessage(event.chatComponent.formattedText)
 
-        event.chatComponent = StringUtils.replaceIfNeeded(event.chatComponent, newMessage) ?: return
+        // TODO apply formattings correctly
+//         event.chatComponent = StringUtils.replaceIfNeeded(event.chatComponent, newMessage) ?: return
```

Comment:
fix

Reply:
Not sure what this was supposed to mean, tbh.